server/README.md

```diff
@@ -105,12 +105,15 @@ otherwise it will append the default port 5432, again to the address without por
 - `DB_USER_FILE`: Specify user to authenticate to the database via file input (alternative to `DB_USER`).
 - `DB_PASSWORD`: Specify user's password to use to authenticate to the database (optional, default is ``).
 - `DB_PASSWORD_FILE`: Specify user's password to use to authenticate to the database via file input (alternative to `DB_PASSWORD`).
+- `BIND_OPTS`: Replaces default binding options, here's an example: `-Djboss.bind.address=10.0.0.1 -Djboss.bind.address.private=10.0.0.2`
 
 ### MySQL Example
 
 #### Create a user defined network
 
     docker network create keycloak-network
+    
+NOTE: When using `Podman` please make sure you create network with `root` permissions. This step is required prior to running any Keycloak container. Running Keycloak container without creating a network will result with an error.
 
```

Comment:
@slaskawi but when you create a network with `root` permissions and try to use it as regular user. You end up with something like:
```
Error: error running container create option: cannot use CNI networks with rootless containers
```
Am I missing something?

Reply:
No, I think that's fine. It seems Podman requires root permissions to mess up with networking.